COMPONENT_VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
Based on discussion with Andrew we will most likely move to `stable` instead of a release number.  But leave this for now until I experiment in another repo and see what happens if we omit it or change to stable